packages/server/postgres/migrations/1709551949071_makeAllTemplatesFree.ts

```diff
@@ -0,0 +1,62 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(
+    `
+    UPDATE "MeetingTemplate"
+      SET "isFree" = true
+      WHERE "orgId" = 'aGhostOrg'
+   `
+  )
+
+  await client.end()
+}
+
+const PREV_FREE_TEMPLATE_IDS = [
```

Comment:
-1 We're still missing more free templates: https://github.com/ParabolInc/parabol/blob/master/packages/server/postgres/migrations/1698831891828_moreFreeRetroTemplates.ts

There are probably others as well. I'd rather skip the down migration altogether than making more templates paid than necessary.

Reply:
+1 This can now be removed